modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
+
+.Procedure
+
+. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
```

Comment:
```suggestion
. You can set `manifests.kustomizePaths` to `++"++</opt/alternate/path>++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++</opt/alternative/path.d/>++*"++` in the configuration file for a glob pattern.
```
I'm not sure which parts of that entry are required, and which are user-specific, but we usually <user-input> format that.

Reply:
Thanks @ShaunaDiaz i was forgot about that as an option.